<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentPageLayout</class>
 <widget class="QWidget" name="ComponentPageLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelPageSize">
     <property name="text">
      <string>Size:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="AdvancedSpinBox" name="numberHeight"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelOrientation">
     <property name="text">
      <string>Orientation:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelUnits">
     <property name="text">
      <string>Units:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QToolButton" name="buttonPortrait">
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonLandscape">
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="AdvancedSpinBox" name="numberWidth"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelHeight">
     <property name="text">
      <string>Height:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="UnitComboBox" name="comboUnits"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboPageSize"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelWidth">
     <property name="text">
      <string>Width:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxMoveObjects">
     <property name="text">
      <string>Move Objects with their Page</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdvancedSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>advanced_spinbox/advanced_spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>UnitComboBox</class>
   <extends>QComboBox</extends>
   <header>_combobox/unit_combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
